### PR TITLE
Update docstring for _get_console method of QtViewer class

### PR DESCRIPTION
**Pull request by GenevieveBuckley**
# Description
The docstring for the `_get_console` method of the `QtViewer` class  is unfinished. I've made some minor changes to tidy it up.

## Context
I had a conversation with Juan about documenting return values. Most of the other methods in the QtViewer class are also pretty sparse on information (so another PR would be welcome), but this is the only obviously unfinished one (so that's why this PR addresses it specifically).

### CODE REVIEW

File: napari/_qt/qt_viewer.py [hunk 1]
```diff
@@ -540,15 +540,16 @@ def console_backlog(self):
         return self._console_backlog
 
     def _get_console(self) -> Optional[QtConsole]:
-        """
-        Function for setup console.
+        """Function to setup console.
 
         Returns
         -------
+        console : QtConsole
```
Comment (lucyleeow) [inline napari/_qt/qt_viewer.py]: nitpick
```suggestion
        console : Optional[QtConsole]
```

File: napari/_qt/qt_viewer.py [hunk 2]
```diff
@@ -540,15 +540,16 @@ def console_backlog(self):
...
+            The napari console.
 
         Notes
         _____
-        extracted to separated function for simplify testing
+        _get_console extracted to separated function for simplify testing
```
Comment (lucyleeow) [inline napari/_qt/qt_viewer.py]:
```suggestion
        _get_console extracted to separate function to simplify testing.
```
nitpick

**Review by lucyleeow — APPROVED** [review]
Thanks!

**Review by Czaki — APPROVED** [review]

File: napari/_qt/qt_viewer.py [hunk 3]
```diff
@@ -540,15 +540,16 @@ def console_backlog(self):
         return self._console_backlog
 
     def _get_console(self) -> Optional[QtConsole]:
-        """
-        Function for setup console.
+        """Function to setup console.
 
         Returns
         -------
+        console : Optional[QtConsole]
```
Comment (Czaki) [inline napari/_qt/qt_viewer.py]: in other docstrings in this file, we have "type or None"

So maybe, for consistency:
```suggestion
        console : QtConsole or None
```

We should decide how to format docstrings

Comment (GenevieveBuckley) [reply napari/_qt/qt_viewer.py]: > We should decide how to format docstrings

FYI there's nothing necessary to decide, the convention here is to use numpy-style docstrings.

**Review by jni — APPROVED** [review]

**Pull request merged**